### ISSUE TITLE
Bug in the description of RMSNorm

### ISSUE THREAD

**Issue by LaurenceA** [pytorch/pytorch#139274]
### 📚 The doc issue

The formula on:

https://pytorch.org/docs/stable/generated/torch.nn.RMSNorm.html#rmsnorm

Is:

.. math::
        y = \frac{x}{\sqrt{\mathrm{RMS}[x] + \epsilon}} * \gamma

i.e. "root mean square" lives inside square root, which is wrong.  We probably want:

.. math::
        y = \frac{x}{\sqrt{\mathrm{MS}[x] + \epsilon}} * \gamma

where MS is "mean-square".

**Comment by awgu** [comment]
It is fixed in main: https://pytorch.org/docs/main/generated/torch.nn.modules.normalization.RMSNorm.html
